### ISSUE TITLE
CemU crashed instantly, after I add the Update and the DLCs!

### ISSUE THREAD

**Issue by Casseopaya** [Tsume/Maple-Tree#133]
When I launch the Games (MK8, Maker), CemU (actually newest Version) works fine.

Then I update the Games with MapleSeed2 (Newest version)

Finally, I will load the updated Games in CemU, but CemU crashed instantly!

Need help!
~ Casseopaya, from Germany
![unbenannt](https://user-images.githubusercontent.com/33157580/32110384-b3df3772-bb37-11e7-9ccc-8543cd52296c.PNG)

**Comment by Tsume** [comment]
This happens with 2.1.1.17b?

Could you provide a step by step of what you did to get cemu to crash? As much detail as possible would be most helpful.

**Comment by Casseopaya** [comment]
Ok tried it! 😥

I downloaded the Games Mario Kart 8 and Super Mario Maker via WiiU USB Helper for CemU. (version 1.10.0f)
I am not update the Games via WiiU USB Helper.
I updated the Games via MapleSeed2 (2.1.1.17b)
The Download is finished. 
I open CemU and run on of these Games.
And then I get the issue from the pic!  👆

Hope that was helpfull (?) 😂😂 (My English is not so good, all English teacher in Germany are sucking)
